src/frontends/tensorflow_common/src/op/rsqrt.cpp

```diff
@@ -17,10 +17,52 @@ namespace op {
 OutputVector translate_rsqrt_op(const NodeContext& node) {
     default_op_checks(node, 1, {"Rsqrt", "RSQRT"});
```

Comment:
```suggestion
    default_op_checks(node, 1, {"Rsqrt", "RSQRT"}, true);
```
Please double-check that layer test is passing locally. Now it should not be passing.